fw/http_frame.c

```diff
@@ -1717,6 +1717,7 @@ tfw_h2_frame_process(void *c, TfwFsmData *data)
 	default:
 		T_WARN("Unrecognized return code %d during HTTP/2 frame"
 		       " receiving, drop frame\n", r);
+		// fallthrough
```

Comment:
We don't use C++ style comments, always `/* ... */`

Reply:
Ok